README.md

```diff
@@ -0,0 +1,109 @@
+
+- As a user, I want to have a map of all of the resorts in my general vicinity so that I can pick a place that I want to go. <br/> 
+
+- As a user, after creating the group, I want to have the ability to edit group details (potentially remove users from group.)<br/>
+
+
+#### Developers:
+
+- As a developer, I want the flow of the website to make sense, such that the user experience is enjoyable. <br/>
+
+- As a developer, I want our database and view progression to have parallel structures so that we can streamline organization of both. <br/>
+
+- As a developer, I want to use authorization in my website so that non-registered users don’t have access to the things that they shouldn’t. <br/>
+
+- As a developer, I want an object representing representing each user to exist in the database so that I can access and show user profile data. <br/>
+
+- As a developer, I want the information in the database to be relationally organized so that related data is easy to find. <br/>
+
+- As a developer I want the API calls to Google Maps and NOAA to successfully render a map with  nearby resorts and to return weather data. <br/>
+
+- As a developer, I want to make an API call to Cordova to ascertain my phones GeoCode location to provide Google Maps and NOAA with correct location data. <br/>
+
+#### Stretch Goals:
+- As a user, I want to be able to add friends to the app so that I can connect with people whom I’ve met at various meetups. <br/>
+
+- As a dev, I want to give the group creators admin privileges. This will enable them to remove anyone from the group who is abusing the site. <br/>
+
+- As a dev, I want to drop a pin on the Google Maps iFrame to provide a visual representation of which groups are snowboarding on specific passes. 
+
+
+### Workflow: 
+
+Each of our teammates checked out a new feature branch when building out a new component of our web app. Once we were done with a feature, we pushed our work to a remote development branch for staging. Before merging, we performed a code review with at least one other team member before merging to our development branch. After our development branch was updated according to a specific milestone, we then merged the development branch to master. 
+
+During development, we deployed our website on heroku from our development branch. Upon completion of the first iteration of our project, we then merged everything to master. We then switched the deploy branch from development to master on heroku as well. 
+
+### Testing:
+To test this app, follow these instructions:
+1. Ensure the auto-create line in application.properties is uncommented, and run the app itself at least once to create the proper database table structure and remove reset all local data.
+2. Run the provided resorts.sql and teams_and_users.sql files to create the dummy data needed for testing
+3. Run the tests through your prefered IDE, or with `./gradlew test` in your console.

```

Comment:
step 3 needs to include "re comment out the auto-create line in application.properties"